TERMSOFSERVICE.md

```diff
@@ -150,13 +182,12 @@ other legal and contractual rights with regard to payment delay shall remain
 unaffected.
 
 6.9. Should you fall into arrears with your payment, you must expect to be
-charged 8% interest for late payment (should you be an entrepreneur) or 5%
-respectively (should you be a consumer) above the base interest rate. Default
-interest will be charged, even without a written warning, if the payment is not
-made at the agreed date. We reserve the right to raise a claim for a higher
-value of compensation for such delay.
+charged 8% interest for late payment (should you be an entrepreneur) above the
```

Comment:
Removed mention of consumers here, which are not allowed to sign up to Tideways as per ToS §4.2